src/impls.rs

```diff
@@ -337,7 +337,7 @@ impl<T> TypeInfo for PhantomData<T> {
         // Fields of this type should be filtered out and never appear in the type graph.
         Type::builder()
             .path(Path::prelude("PhantomData"))
-            .docs(&["PhantomData placeholder, this type should be filtered out"])
+            .docs(["PhantomData placeholder, this type should be filtered out"])
```

Comment:
Note that this is not a required change, just a clippy suggestion since arrays implement `IntoIter`